apiserver/params/crossmodel.go

```diff
@@ -246,11 +246,12 @@ type RemoteEntityIdResults struct {
 // RemoteRelation describes the current state of a cross-model relation from
 // the perspective of the local model.
 type RemoteRelation struct {
-	Life               Life           `json:"life"`
-	Id                 int            `json:"id"`
-	Key                string         `json:"key"`
-	LocalEndpoint      RemoteEndpoint `json:"local-endpoint"`
-	RemoteEndpointName string         `json:"remote-endpoint-name"`
+	Life                 Life           `json:"life"`
+	Id                   int            `json:"id"`
+	Key                  string         `json:"key"`
+	LocalApplicationName string         `json:"local-application-name"`
+	LocalEndpoint        RemoteEndpoint `json:"local-endpoint"`
```

Comment:
RemoteEndpoint is different to Endpoit. The term "local" refers to which model the data belongs in. Models have both Endpoit and RemoteEndpoint data. I'll add a comment